### PR TITLE
2024 hu.m3u

### PR DESCRIPTION
**Pull request by 1134214**

### CODE REVIEW

**Comment by jianghongc** [comment]
没显示

File: streams/ro.m3u [hunk 1]
```diff
@@ -15,6 +15,10 @@ http://s5.alfaomega.tv:1935/alfaomega/smil:alfaomegatv/playlist.m3u8
 https://transmisiuni.ro/media/oracatolica/playlist.m3u8
 #EXTINF:-1 tvg-id="Antena1.ro",Antena 1 (1080p)
 https://live1ag.antenaplay.ro/live_a1ro/live_a1ro.m3u8
+#EXTINF:-1 tvg-id="Antena3CNN.ro",Antena 3 CNN
+https://live3vox.antenaplay.ro/a3free/a3free.m3u8
+#EXTINF:-1 tvg-id="AntenaStars.ro",Antena Stars (1080p)
+http://byasmartv.net:8080/live/HIhVbtM9tW8UdZy5k4ymWcaZFDsD0Y/PAgPDVBz6J4U6YN8LBwcfppeU46fq9/9052.ts
```
Comment (BellezaEmporium) [inline streams/ro.m3u:21]: Xtream-Codes, not accepted in here.

File: streams/ro.m3u [hunk 2]
```diff
@@ -128,6 +136,8 @@ https://5ab29cc78f681.streamlock.net/NasulTV/livestream/playlist.m3u8
 http://novapress.ro:2022/live/nova/playlist.m3u8
 #EXTINF:-1 tvg-id="ObservatorNews.ro",Observator News (720p)
 https://live1ag.antenaplay.ro/live_ObservatorNews/live_ObservatorNews.m3u8
+#EXTINF:-1 tvg-id="PartyMix.ro",Party Mix (1080p)
+http://103.145.13.15:25461/live/promo/promo/2.m3u8
```
Comment (BellezaEmporium) [inline streams/ro.m3u:140]: Not quite sure. It's an Xtream-Codes but seems to be owned by the company that has this channel.

**Comment by leoneljimenez40** [comment]
agregar #16305

**Comment by BellezaEmporium** [comment]
After re-verification, none of them works.

**Pull request closed without merging**